### ISSUE TITLE
Error When Opening Settings Menu

### ISSUE THREAD

**Issue by DivertedCircle** [trioderegion/dnd5e-helpers#276]
Getting the following error in the new 4.1.0 release when clicking the 'Open Helper Settings' button under configure game settings. Clicking the button throws the error and does not open the settings window. Running Foundry version 9, build 255 and DnD 5e v1.5.7.

![image](https://user-images.githubusercontent.com/26470570/160488886-d03f6b14-78fb-4ae8-8c9e-8405d85503e5.png)

**Comment by GamerFlix** [comment]
Unable to reproduce can you narrow down how to trigger this? Tested with the current version of each of the relevant modules (see screenshot)
![grafik](https://user-images.githubusercontent.com/62909799/160507954-7ef2f0e6-28f7-4bd4-8b6d-59da433f223c.png)

**Comment by DivertedCircle** [comment]
Playing around with Find the Culprit has revealed the error only occurs when DnD 5e Helpers Cover Expansion is also enabled.
https://github.com/kandashi/dnd5e-helpers-cover-expansion

**Comment by trioderegion** [comment]
Yep, that's a known issue at the moment, I'll be trying to get a patch out today for it.
